src/core/Akka.Streams/Serialization/StreamRefSerializer.cs

```diff
@@ -169,7 +167,7 @@ private ByteString SerializeCumulativeDemand(CumulativeDemand demand) =>
         private ByteString SerializeSequencedOnNext(SequencedOnNext onNext)
         {
             var payload = onNext.Payload;
-            var serializer = _serialization.FindSerializerFor(payload);
+            var serializer = system.Serialization.FindSerializerFor(payload);
```

Comment:
Actual fix. Instead of using a field assigned `Serialization`, we just retrieve it from `_system.Serialization` when we need it.